VideoLocker/src/main/java/org/edx/mobile/module/analytics/ISegmentImpl.java

```diff
@@ -189,7 +190,8 @@ public Properties trackVideoSeek(String videoId,
             skipInterval = formatDoubleValue(skipInterval, 3);
             aEvent.data.putValue(Keys.OLD_TIME, oldTime);
             aEvent.data.putValue(Keys.NEW_TIME, newTime);
-            aEvent.data.putValue(Keys.SEEK_TYPE, Values.SKIP);
+            if (skipSeek){ aEvent.data.putValue(Keys.SEEK_TYPE, Values.SKIP);}
```

Comment:
For me, the main advantage of an enum was seeing the name when looking at the code, instead of just ``true`` and ``false``, which means I have to actually look up the method definition to see what the parameter is. It's a small enough thing, that I don't care too much though.